ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -137,7 +139,12 @@ export default function OnboardingMetametrics() {
           {t('onboardingMetametricsSendAnonymize')}
         </li>
         <li>
-          <i className="fa fa-times" />
+          <Icon
+            name={ICON_NAMES.CLOSE}
+            as="span"
+            className="onboarding-metametrics__close"
+            size={Size.inherit}
+          />
```

Comment:
### Before
<img width="683" alt="Screenshot 2023-02-06 at 8 32 51 PM" src="https://user-images.githubusercontent.com/39872794/217010354-06aad60d-a6ce-404c-b048-52cd7f676660.png">

### After
<img width="719" alt="Screenshot 2023-02-06 at 8 44 56 PM" src="https://user-images.githubusercontent.com/39872794/217010364-08170ade-9a65-4a2e-bedd-0008db9b1739.png">

Reply:
@georgewrmarshall is it fine to use `Icon` here?

Reply:
We should also fix the line height ; the "Never" text starts too low

Reply:
Yes! Fine to use `Icon` here. Can add `align-items: baseline` to align icon with text

<img width="1440" alt="Screenshot 2023-02-06 at 2 59 00 PM" src="https://user-images.githubusercontent.com/8112138/217106644-a28d573b-b022-463e-af2f-da3f4a454693.png">

Reply:
Actually I think there is a better way to handle this. If we remove the default marginTop from the Typography component and remove the className from the `Icon` it should align better. Heres an example

```
<li>
          <Icon
            name={ICON_NAMES.CLOSE}
            color={IconColor.errorDefault}
            marginInlineEnd={2}
          />
          {t('onboardingMetametricsNeverCollect', [
            <Typography
              variant={TypographyVariant.Span}
              key="never"
              fontWeight={FONT_WEIGHT.BOLD}
              marginTop={0}
            >
              {t('onboardingMetametricsNeverEmphasis')}
            </Typography>,
          ])}
        </li>
```